_docs/README.md

```diff
@@ -153,7 +156,8 @@ boilerplate --template-folder ~/templates --output-folder ~/output --var-file va
 #### The boilerplate.yml file
 
 The `boilerplate.yml` file is used to configure `boilerplate`. The file is optional. If you don't specify it, you can
-still use Go templating in your templates, but no variables or dependencies will be available.
+still use Go templating in your templates so long as you specify the `--missing-config-action ignore` option, but no
+variables or dependencies will be available.
```

Comment:
Technically, couldn't you use boilerplate's plugins (not sure the right word) with literals as well?